man/systemd.time.xml

```diff
@@ -154,6 +154,8 @@
     <programlisting>  Fri 2012-11-23 11:12:13 → Fri 2012-11-23 11:12:13
       2012-11-23 11:12:13 → Fri 2012-11-23 11:12:13
   2012-11-23 11:12:13 UTC → Fri 2012-11-23 19:12:13
+     2012-11-23T11:12:13Z → Fri 2012-11-23 19:12:13
+   2012-11-23T11:12+02:00 → Fri 2012-11-23 17:12:13
```

Comment:
this needs some more docs btw. i.e. at some appropriate place we should mention that RFC3339 syntax is also supported, in it's strictest form.

Reply:
Added